src/analysis/site_selection.jl

```diff
@@ -0,0 +1,56 @@
+using NamedDims, AxisKeys
+using ADRIA: ResultSet, n_locations
+
+
+"""
+    seeded_sites_frequency(rs::ResultSet,scens::NamedTuple, log_type::String)::NamedDimsArray
+
+Count frequency of seeded sites for scenarios satisfying a condition.
+
+# Arguments
+- 'rs' : ResultSet
+- `scens` : contains scenario ids for scenarios satisfying the condition of interest.
+- 'log_type` : "seed", "shade" or "fog" indicating the intervention log to use in calculating frequencies.
+
+# Returns
+NamedDimsArray(:locations,:rcps)
+
+# Example
+```julia
+using ADRIA, Statistics
+
+rs = ADRIA.load_results("some result set")
+
+tac = ADRIA.metrics.scenario_total_cover(rs)
+rsv = ADRIA.metrics.scenario_rsv(rs)
+
+# Create matrix of mean scenario outcomes
+mean_tac = vec(mean(tac, dims=1))
+mean_sv = vec(mean(rsv, dims=1))
+y = hcat(mean_tac, mean_sv)
+
+# Find all pareto optimal scenarios where all metrics >= 0.9
+rule_func = x -> all(x .>= 0.9)
+robust = ADRIA.analysis.find_robust(rs, y, rule_func, [45, 60])
+
+# find site selection frequencies for all robust scenarios
+robust_selection_frequencies = intervention_frequency(rs, scen_indices)
```

Comment:
Wondering how this is being used on your end? I hit an undefined error, and I can't see this file being included anywhere - doesn't seem to be part of the ADRIA package?

Reply:
sorry,I just realised I was supposed to ask you about this in the PR and forgot to- should the import of site_selection.jl be in the ADRIA.jl file or the analysis module or both?

Reply:
Thanks, so analysis files only need to be imported in the analysis module?

Reply:
Yes - functions/methods should be grouped/compartmentalised by modules according to their purpose. Their files are grouped by folders where appropriate.

You should see this if you look at the package directory structure. You should be able to intuit what lives where, and how it all relates to each other. For example, all code for analysis live together, ditto for file handling, etc.

At the top level, we only need to import the sub-modules (and later, package extensions).
Then the user only needs to import ADRIA or the sub-modules as necessary.

Otherwise you get into a situation where it's a messy free-for-all.